apps/fluent-tester/src/FluentTester/TestComponents/Button/ButtonSizeTestSection.tsx

```diff
@@ -11,12 +11,12 @@ export const ButtonSizeTest: React.FunctionComponent = () => {
     viewBox: '0 0 500 500',
   };
   const svgIconsEnabled = ['ios', 'macos', 'win32', 'android'].includes(Platform.OS as string);
-
+  const CustomButton = Button.customize({ iconColor: 'yellow' });
```

Comment:
1. Could we put this in a separate test (maybe the HOC page) so that this test stays purely about showing sizes of buttons?
2. Creating customized components should be outside of any component render functions, otherwise the component is recreated on every render.